domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/UserRoleRequest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import javax.validation.constraints.NotEmpty;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class UserRoleRequest {
+    @NotEmpty
+    @JsonProperty("role_name")
```

Comment:
Make it shorter into just a `role`. Then follow up by renaming `roleName` into `role`.